src/main/java/io/debezium/connector/informix/InformixValueConverters.java

```diff
@@ -41,20 +51,106 @@ public InformixValueConverters(DecimalMode decimalMode, TemporalPrecisionMode te
 
     @Override
     public SchemaBuilder schemaBuilder(Column column) {
+        logger.debug("Building schema for column {} of type {} named {} with constraints ({},{})",
+                column.name(),
+                column.jdbcType(),
+                column.typeName(),
+                column.length(),
+                column.scale());
+
         switch (column.jdbcType()) {
+            case Types.NUMERIC:
+            case Types.DECIMAL:
+                return getNumericSchema(column);
             default:
-                return super.schemaBuilder(column);
+                SchemaBuilder builder = super.schemaBuilder(column);
+                logger.debug("JdbcValueConverters returned '{}' for column '{}'", builder != null ? builder.getClass().getName() : null, column.name());
+                return builder;
+        }
+    }
+
+    private SchemaBuilder getNumericSchema(Column column) {
+        if (column.scale().isPresent()) {
+
+            if (column.scale().get() == 255 && decimalMode == DecimalMode.PRECISE) {
```

Comment:
Could we move the `255` to a private static final class variable with a descriptive name to clarify precisely what 255 means?  The value itself is less important than its purpose.  There is another instance of this in a method below around like ~108.

Reply:
Certainly. 
I'm unsure if Informix actually encodes scale as a signed byte when everyone else thinks it is a wider field, and sends -1 to signal variable scale or if this is just an arbitrary number. I noticed that DataGrip "successfully" interpret these fields has having an actual scale of 255...